CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+3.5.0
+---
+
+* Add the commands `sign`, `add-sig`, and `combine-sigs` for working with signatures (#759)
```

Comment:
Following the format for other entries, please include all fixes except possibly CI- or tooling-only ones

```
5832234 * master origin/master Pact offline signing improvements (#759)
7c493de * GitHub ci (#761)
5f19f4b * Fix ghcjs build (#752)
43f2bbb * Tiny instance for ChainId  (#756)
fe8fce1 * Decode StackFrame JSON output (#755)
0aa3ea9 * CommandResult golden coverage (#754)
5051460 *   Merge branch 'master' of github.com:kadena-io/pact
        |\  
593938a | * Use cffi for aeson (upstreamed from chainweb) (#749)
e95bd01 * | updated pact version in pact-functions.md
        |/  
81c12bf * Adds gas logs to eval result (#748)
```